assets/css/src/amp-mobile-version-switcher.css

```diff
@@ -1,6 +1,12 @@
+#amp-mobile-version-switcher {
+	position: absolute;
+	width: 100%;
+	left: 0;
+	z-index: 100;
```

Comment:
This property is needed to override the `:before` pseudo-element extending the `body` in Twenty Sixteen.